pyproject.toml

```diff
@@ -0,0 +1,108 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-morphologicalcontourinterpolation"
+version = "2.0.0"
+description = "A morphology-based approach for interslice interpolation of anatomical slices from volumetric images."
+readme = "README.md"
+license = {file = "LICENSE"}
+authors = [
+    { name = "Dženan Zukić", email = "community@itk.org" },
+]
+keywords = [
+    "itk",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: Android",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Programming Language :: C++",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Scientific/Engineering :: Medical Science Apps.",
+    "Topic :: Software Development :: Libraries",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "itk == 5.4.*",
```

Comment:
Do itk modules also get listed here?   e.g., itk-rleimage and itk-morphologicalcontourinterpolator?

Reply:
If I remember correctly, `RLEImage` is a test-time dependency.

Reply:
Yes, it is a test dependency:
https://github.com/KitwareMedical/ITKContourInterpolation/blob/v1.0.1/itk-module.cmake#L11